<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QPushButton" name="backButtonS">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumFiles">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音声ファイル数</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>711</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
